package.json

```diff
@@ -100,6 +100,7 @@
     "react-lottie": "^1.2.3",
     "react-markdown": "^4.3.0",
     "react-motion": "^0.5.2",
+    "react-prismazoom": "^2.1.1-rc1",
```

Comment:
could you use v3 instead not a fan of locking to rc versions for prod :sweat_smile:

Reply:
also correct me if i'm wrong but this doesn't seem so hard to do natively on desktop the whole mousewheel to zoom and transform scale :sweat_smile: if you do so move that logic towards the ui lib so we can reuse it

Reply:
Great point on the version 😅, bumped it to 3.0.0.

Concerning doing it ourselves. I don't think it'd be too hard, but still will take a bit of time. Will keep that in mind to maybe tackle it as technical task when I have some time.